mp/src/game/server/momentum/mom_player.cpp

```diff
@@ -1033,6 +1059,12 @@ void CMomentumPlayer::OnZoneExit(CTriggerZone *pTrigger)
     case ZONE_TYPE_CHECKPOINT:
         break;
     case ZONE_TYPE_START:
+        if (g_pGameModeSystem->GameModeIs(GAMEMODE_SJ))
+        {
+            // Re-enable charge on start zone exit
+            static ConVarRef mom_sj_charge_enable("mom_sj_charge_enable");
+            mom_sj_charge_enable.SetValue(1);
```

Comment:
I don't like this. This convar is going to be changed by players, forcing the value back to on/off when the player wants it as another value is bad.

Why not check the player inside the launcher, and see if they're in the start zone to prevent it? You already do that in the validator for the convar.